src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": "0.1",
     "requires": {
-        "cockpit": "138"
+        "cockpit": "137"
```

Comment:
Yeah I'm not to sure about that either. The alternative would be to use cockpit from the copr for this release.